### PR TITLE
Pba 2553c

### PR DESCRIPTION
**Pull request by Matthew-C**
fixed pause on more options and some other stuff. please let me know if you see any problems.

### CODE REVIEW

File: sdk/react/ooyalaSkinCore.js [hunk 1]
```diff
@@ -114,7 +115,7 @@ OoyalaSkinCore.prototype.onOptionDismissed = function() {
 };
 
 OoyalaSkinCore.prototype.handlePress = function(n) {
-  this.skin.setState({lastPressedTime: (new Date).getTime()});
+  console.log("handlePress+buttonName:"+n);
```
Comment (soapyigu) [inline sdk/react/ooyalaSkinCore.js]: Do we really need this log in the product?

Comment (Matthew-C) [inline sdk/react/ooyalaSkinCore.js]: removed.

File: sdk/react/widgets/VideoViewPlayPause.js [hunk 2]
```diff
@@ -50,100 +49,109 @@ var VideoViewPlayPause = React.createClass({
...
+      this.state.widget.animationOpacity.setValue(this.props.showButton ? 1 : 0);
+      this.state.play.animationOpacity.setValue(this.props.playing ? 0 : 1);
+      this.state.pause.animationOpacity.setValue(this.props.playing ? 1 : 0);
     }
   },
 
-  isInitialVideoPlay: function() {
-    return (!this.props.isStartScreen && this.props.playhead == 0);
+  componentDidMount: function () {
+    console.log("componentdidmount");
```
Comment (soapyigu) [inline sdk/react/widgets/VideoViewPlayPause.js]: This has to be removed

Comment (Matthew-C) [inline sdk/react/widgets/VideoViewPlayPause.js]: removed.

**Pull request merged**